### ISSUE TITLE
failed pdf export

### ISSUE THREAD

**Issue by saracarl** [benwbrum/fromthepage#2723]
```
! Extra }, or forgotten \endgroup.
\UL@stop ...z@ \else \UL@putbox \fi \else \egroup 
                                                  \egroup \UL@putbox \fi \if...
l.354 Woodruff}}
                 was baptized for in the font in\\
Output written on /tmp/tex2pdf.-f53306599b3c111f/input.pdf (16 pages).
Transcript written on /tmp/tex2pdf.-f53306599b3c111f/input.log.
```

Woodruff.

**Comment by saracarl** [comment]
From:
```
                Exporting work  48160   Journal (December 31, 1842–December 31, 1842)
pandoc --from markdown+superscript -o /home/fromthepage/deployment/releases/20210731180423/public/printable/20210816190306/wilford_woodruff_journal_january_1841_december_1842_section_10_20210816190306.pdf
 /home/fromthepage/deployment/releases/20210731180423/public/printable/20210816190306/wilford_woodruff_journal_january_1841_december_1842_section_10_20210816190306.md --pdf-engine=xelatex --verbose > /hom
e/fromthepage/deployment/releases/20210731180423/public/printable/20210816190306/wilford_woodruff_journal_january_1841_december_1842_section_10_20210816190306.log 2>&1
```

**Comment by saracarl** [comment]
This page seems to be the problem:
https://fromthepage.com/woodruff/woodruffpapers/wilford-woodruff-journal-january-1841-december-1842-section-10/transcribe/1592491

The only thing "odd" I can see here is the highlight tag that wraps the line.

**Comment by saracarl** [comment]
Here's the markdown for that section; we should try pandoc'ing it separately to see if it works:
```
The following are the names of the persons\
that [Phebe W.
Woodruff](#article-106888 "Phebe Whittemore Carter Woodruff") has been
baptized\
for in the font in the
[Temple](#article-91275 "Nauvoo Temple, Nauvoo, Hancock County, Illinois")
of the Lord in\
Nauvoo in 1842 \^under the hands of Wilford Woodruff\^
```

**Comment by saracarl** [comment]
Ran that subset of markdown in pandoc without error.

**Comment by saracarl** [comment]
The following are the names of the\
Dead that Wilford Woodruff ~~& [Phebe W\
Woodruff](#article-106888 "Phebe Whittemore Carter Woodruff")~~ was
baptized for in the font in\
the house of the Lord in
[Nauvoo](#article-86735 "Nauvoo, Hancock County, Illinois"), in 1842

**Comment by saracarl** [comment]
Ok.  I was using the wrong page (they are similar).  The text on page 7 is:
```
The following are the names of the 
Dead that Wilford Woodruff <hi rend="str">& [[Phebe Whittemore Carter Woodruff|Phebe W 
Woodruff]]</hi> was baptized for in the font in 
the house of the Lord in [[Nauvoo, Hancock County, Illinois|Nauvoo]], in 1842
```
look for "Dead" to differentiate it from the others.

Once we found that the markdown was as follows:
```
The following are the names of the\
Dead that Wilford Woodruff ~~& [Phebe W\
Woodruff](#article-106888 "Phebe Whittemore Carter Woodruff")~~ was
baptized for in the font in\
the house of the Lord in
[Nauvoo](#article-86735 "Nauvoo, Hancock County, Illinois"), in 1842
```

That causes the same problem when put into the pdf generation command.  We then ran tests.  It succeeded if we:
1)  removed the \ (line break) at the end of the Phebe W line
or
2)  removed the strike-through around Phebe that spans the line break
or
3)  removed the subject link around Phebe that spans the line break

We're not sure we need a \ at the end of Phebe W in the text; it seems to be inserted because of the subject link or the format tagging. 

We looked at the full html export (without lines); see it by following:
https://fromthepage.com/display/read_work?page=2&work_id=48160
and noticed this:
```
Phebe W <span class='line-break'> </span>Woodruff
```
We think the <span> tag here is why we get a \ linebreak by the time we are in markdown for pdf generation.
